packages/react-components/src/components/Survey/TagsSurvey/TagsSurvey.tsx

```diff
@@ -173,25 +173,30 @@ export const _TagsSurvey = (props: _TagsSurveyProps): JSX.Element => {
           placeholder={strings?.tagsSurveyTextFieldDefaultText}
           onChange={(e, v) => {
             if (v) {
+              setCheckedTextFields([...checkedTextFields, issueCategory]);
```

Comment:
If the text field is empty, (i.e. they typed something then deleted it) should it uncheck it again?

Reply:
Chatted with Alex P he said to keep it simple, we are not unchecking it when it's empty